spec/jobs/send_job_application_data_expiry_notification_job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe SendJobApplicationDataExpiryNotificationJob do
+  subject(:job) { described_class.perform }
+
+  let(:organisation) { create(:school) }
+  let(:publisher) { create(:publisher) }
+  let(:two_weeks_less_than_a_year_ago) { Time.zone.now - (1.year - 2.weeks) }
+  let(:vacancy) { create(:vacancy, expires_at: two_weeks_less_than_a_year_ago, publisher: publisher, organisation_vacancies_attributes: [{ organisation: organisation }]) }
+
+  let(:notification) { double(:notification) }
```

Comment:
`instance_double` if you like. It complains when given a method not implemented by Notification.